### ISSUE TITLE
Extension broken in [2.0.4...2.0.6], reverted in 2.0.7

### ISSUE THREAD

**Issue by dispbd** [philfontaine/autolaunch#15]
The extension stopped working after today's update.
If you enable developer tools in VSCODE, then it shows this error:
![Code_5pDvAWG0jb](https://user-images.githubusercontent.com/45291423/121716921-4ba18780-cae9-11eb-8250-36a3e778c3b5.png)
My config settings for running projects:
```javascript
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "_",
      "type": "shell",
      "command": "node",
      "args": [
        "debug.js"
      ],
	  "presentation": {
        "group": "shared"
      }
      "auto": true,
      "problemMatcher": []
    },
    {
      "label": "_d",
      "type": "shell",
      "command": "node",
      "args": [
        "compile.js"
      ],
	  "presentation": {
        "group": "shared"
      }
      "auto": true,
      "problemMatcher": []
    }
  ]
}
```

**Comment by dispbd** [comment]
Temporarily returned to the previous version, so it's not critical.
Thank you for the extension!

**Comment by philfontaine** [comment]
That is very unfortunate, sorry about that. Temporarily published 2.0.7 which reverts the changes (but you can just stay at 2.0.3 in the meantime if you want). 

I'll have to investigate more to find out what broke it, it seems to be a dumb issue with yarn/npm. Plus I can't test it unless I publish since it works fine in dev mode. Uhhh...

Thanks for reporting this.

**Comment by philfontaine** [comment]
Published 2.0.9 which contains the changes I wanted to make initially without the bug.

What happened is that I started updating the project with the latest vscode extension webpack template and used the vscodeignore of that template. Once I realized I didn't want/need webpack, I switched to the other template but forgot to replace the vscodeignore.

The end result was that the extension was matched in the vscodeignore, and the file was therefore not published/accessible (hence the "cannot find module").
